src/Templates/src/templates/maui-mobile/Pages/ManageMetaPage.xaml

```diff
@@ -54,7 +54,7 @@
                 <BindableLayout.ItemTemplate>
                     <DataTemplate x:DataType="models:Category">
                         <Grid ColumnSpacing="{StaticResource LayoutSpacing}" ColumnDefinitions="4*,3*,30,Auto">
-                            <Entry Text="{Binding Title}" Grid.Column="0" SemanticProperties.Description="Title" />
+                            <Entry Text="{Binding Title}" SemanticProperties.Description="{Binding Title}" Grid.Column="0" SemanticProperties.Description="Title" />
```

Comment:
This now has two `SemanticProperties.Description` properties

Reply:
Right, fixed.